app/src/main/res/values/attr_expandable_text_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+	<declare-styleable name="ExpandableTextView">
+		<attr name="etv_MaxLinesOnShrink" format="reference|integer" />
+		<attr name="etv_EllipsisHint" format="reference|string" /><!--default is ..-->
+		<attr name="etv_ToExpandHint" format="reference|string" /><!--"to expand" hint string, default is "Expand"-->
+		<attr name="etv_ToShrinkHint" format="reference|string" /><!--"to shrink" hint string, default is "Shrink"-->
+		<attr name="etv_GapToExpandHint" format="reference|string" /><!--gap between "toExpandHint" and "trimmed text"-->
+		<attr name="etv_GapToShrinkHint" format="reference|string" /><!--gap between "toShrinkHint" and "trimmed text"-->
+		<attr name="etv_EnableToggle" format="reference|boolean" /><!--if toggle the state when click ExpandTextView-->
+		<attr name="etv_ToExpandHintShow" format="reference|boolean" /><!--if show "toExpandHint"-->
+		<attr name="etv_ToShrinkHintShow" format="reference|boolean" /><!--if show "toShrinkHint"-->
+		<attr name="etv_ToExpandHintColor" format="reference|color" /><!--text color of "toExpandHint"-->
+		<attr name="etv_ToShrinkHintColor" format="reference|color" /><!--text color of "toShrinkHint"-->
+		<attr name="etv_ToExpandHintColorBgPressed" format="reference|color" /><!--background color if "toExpandHint" pressed-->
+		<attr name="etv_ToShrinkHintColorBgPressed" format="reference|color" /><!--background color if "toShrinkHint" pressed-->
+		<attr name="etv_InitState" format="enum"><!--init state, default is shrink-->
+			<enum name="shrink" value="0"/>
+			<enum name="expand" value="1"/>
+		</attr>
+	</declare-styleable>
+
+</resources>
```

Comment:
Why do we need to add this? I don't really understand the chinese on the libraries' page.